dWorldServer/WorldServer.cpp

```diff
@@ -267,36 +267,26 @@ int main(int argc, char** argv) {
 
 	// pre calculate the FDB checksum
 	if (Game::config->GetValue("check_fdb") == "1") {
-		static const std::vector<std::string> aliases = {
```

Comment:
This removes looking via aliases?

Reply:
cdserver.fdb is not used by anyone afaik and CDClient.fdb is covered by case insensitive searching of file paths.  I dont see a reason to keep the aliases anymore

Reply:
Makes sense